src/components/list/list.scss

```diff
@@ -186,3 +186,15 @@ $list-border-radius: 0.375rem; // 6px
         }
     }
 }
+
+:not(.mdc-list--non-interactive) {
+    > {
+        :not(.mdc-list-item--disabled).mdc-list-item {
```

Comment:
I just wanted to double check that you hadn't forgotten about the disabled state, but since you specifically targeted `:not(.mdc-list-item--disabled)`, I assume you left the styling for disabled the way it was on purpose 🙂

https://user-images.githubusercontent.com/154725/114364594-d57cb500-9b79-11eb-9861-ea9b15651fbc.mov